server-phoenix/lib/helios_web/schema/queries/location.ex

```diff
@@ -0,0 +1,12 @@
+defmodule HeliosWeb.Schema.Queries.Location do
+  use Absinthe.Schema.Notation
+
+  alias HeliosWeb.Schema.Resolvers.Location
+
+  object :location_queries do
+    field :location, :location do
+      arg(:city_name, :string)
+      resolve(&Location.location/3)
```

Comment:
Nit: I'm pretty sure these resolve method calls don't exist in this commit but its not that big of deal.

Reply:
Yeah.... I guess they could be in the resolvers commit or type definitions.